dune

```diff
@@ -6,7 +6,8 @@
  (ireport (flags :standard -rectypes -w -9-27-40+60)
           (ocamlopt_flags :standard -O3 -unbox-closures -inlining-report))
  (ocaml408
-  (flags :standard -strict-sequence -strict-formats -short-paths -keep-locs -rectypes -w -9-27+40+60 -warn-error -5 -alert --deprecated)))
+  (flags :standard -strict-sequence -strict-formats -short-paths -keep-locs -rectypes -w -9-27+40+60 -warn-error -5 -alert --deprecated
+         -ccopt -Dcaml_signals_are_pending=caml_signals_might_be_pending)))
```

Comment:
Not sure if this will actually work, I wanted to put a c_flags entry but that's not allowed (ocaml/dune#1124 / ocaml/dune#1073)